modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -237,10 +237,12 @@ export default {
       }
     },
     getInitialSubmitRequirementsData() {
-      return Object.fromEntries(this.requirements.filter(r => r.phase !== 'afterPasswordVerified').map(r => ([
-        r.name,
-        r.value
-      ])));
+      return Object.fromEntries(this.requirements
+        .filter(r => r.phase !== 'afterPasswordVerified' || !r.done)
```

Comment:
This is going to return all requirements that are not "done," even if they *are* `afterPasswordVerified`. Does that make sense? It seems to me like you probably wanted `&&` here. Also please add parens around the `!==` clause for readability (assuming I have the intended grouping right, that is).

Reply:
I have this error message when trying to login with `&&` instead of `||`:

```
POST /api/v1/@apostrophecms/login/login: 

invalid: The hCaptcha token was missing while verifying login.
    at Object.error (/Users/anttarla/Dev/apostrophecms/apostrophe/modules/@apostrophecms/error/index.js:29:23)
    at Object.verify (/Users/anttarla/Dev/apostrophecms/test-project/node_modules/@apostrophecms/login-hcaptcha/index.js:19:31)
    at Object.initialLogin (/Users/anttarla/Dev/apostrophecms/apostrophe/modules/@apostrophecms/login/index.js:625:33)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async /Users/anttarla/Dev/apostrophecms/apostrophe/modules/@apostrophecms/module/index.js:180:30
```